velox/connectors/hive/HiveDataSink.cpp

```diff
@@ -806,6 +806,8 @@ uint32_t HiveDataSink::appendWriter(const HiveWriterId& id) {
               .stats = ioStats_.back().get(),
           }),
       options);
+  addThreadLocalRuntimeStat(
```

Comment:
Shall we report once for each table write operator so we can see the distribution better?